### PR TITLE
cmd: Add 'version' option

### PR DESCRIPTION
**Pull request by daissi**
This 'version' option prints the debos version. Fix: #430

### CODE REVIEW

File: cmd/debos/debos.go [hunk 1]
```diff
@@ -110,6 +113,11 @@ func main() {
 		}
 	}
 
+	if options.Version {
+		fmt.Printf("debos %s\n", Version)
```
Comment (obbardc) [inline cmd/debos/debos.go:117]:
```suggestion
		fmt.Printf("debos v%s\n", Version)
```

Comment (daissi) [reply cmd/debos/debos.go]: Thank you! Fixed :)

**Comment by obbardc** [comment]
Can you do the same thing for fakemachine ? There's an open issue, i think.

**Comment by daissi** [comment]
> Can you do the same thing for fakemachine ? There's an open issue, i think.

Sure, I will do it tomorrow.

**Pull request merged**